Jenkinsfile.deploy

```diff
@@ -129,6 +247,55 @@ void runMaven(String goals, String directory, boolean skipTests = false, List pr
         mvnCmd += " ${extraArgs}"
     }
     dir(directory) {
-        maven.runMavenWithSubmarineSettings(mvnCmd, skipTests)
+        if(isRelease()){
+            // In case of release, we already have the settings.xml
+            maven.runMaven(mvnCmd, skipTests, ['-fae'])
+        } else {
+            maven.runMavenWithSubmarineSettings(mvnCmd, skipTests)
+        }
+    }
+}
+
+void updateKogitoVersion(String newVersion) {
+    sh "mvn versions:set-property -Dproperty=version.org.kie.kogito -DnewVersion=$newVersion -DallowSnapshots=true -DgenerateBackupPoms=false"
```

Comment:
May be worth moving to a shared library (after parametrization).

Reply:
I do agree